vocabularies/voc4cat/0000269.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0000269>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0000269"^^xsd:token ;
+    skos:altLabel "continuous flow chemistry"@en ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0000181> ;
+    skos:definition "A chemical reaction that is performed in a continuous flow stream of reagents."@en ;
```

Comment:
The definition is for a process. It should be adapted or the concept should be moved to processes.